### PR TITLE
Use read replica for dashboard

### PR DESCRIPTION
**Pull request by insacuri**
# What and why?

The previous version of the reporting app used the reporting database.  It was slow and difficult to maintain as the way to query the database was done via one giant multi database spanning sql query.

This new way uses the read replicas to access only the databases it needs (party and case) which makes it more obvious what is accessing what (for context, the reporting database is a database that uses a foreign data wrapper to treat all of our 10-ish databases for our services as 1 big database).  As this is the only thing using it, it also allows us to remove the reporting database at some point too.

The way the dashboard and the spreadsheets are generated should be easier to maintain as as now make a few smaller database calls to get what we need before compiling that data into something usable.  It should also be easier to test.

This PR will be the first of 2 parts to keep it manageable.  This one gets the functionality implemented, config set and the code looking pretty good.  There will be another one that adds a few more unit tests, docstrings and tidies up the functions to make sure their inputs and outputs make sense

# How to test?

- Deploy environment and run test to seed data
- Either:
    - run locally (using `ras pf` to port forward the database) and hit the endpoints using postman or curl. or
    - Deploy to your k8s environment, but then modify the deployment to get the database uri's in the right state.  Because spinnaker deploys the latest helm chart, it's tricky to get it to work

# Trello

### CODE REVIEW

File: rm_reporting/controllers/case_controller.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        "FROM casesvc.casegroup "
+        "WHERE collection_exercise_id = :collection_exercise_id "
+        "AND sample_unit_ref NOT LIKE '1111%'"
+    )
+
+    return case_engine.execute(case_query, collection_exercise_id=collection_exercise_id).all()
+
+
+def get_all_business_ids_for_collection_exercise(collection_exercise_id: str) -> str:
+    logger.info("About to get all business  ids for collection exercise", collection_exercise_id=collection_exercise_id)
```
Comment (SteveScorfield) [inline rm_reporting/controllers/case_controller.py:69]: Very minor but there is a double space between business and ids.

Comment (insacuri) [reply rm_reporting/controllers/case_controller.py]: A silly mistake ^^  Just pushed a commit to fix it

**Review by SteveScorfield — COMMENTED** [review]
Apart from the minor comment already mentioned, this looks all good to me.

**Pull request merged**